VaadinatorTooling/VaadinatorGenerator/src/main/java/de/akquinet/engineering/vaadinator/mojo/CodeGeneratorMojo.java

```diff
@@ -558,6 +559,10 @@ private static void runVelocity(BeanDescription desc, Map<String, Object> common
 		writer.close();
 	}
 
+	public String toValidJavaClassName(String name) {
```

Comment:
Bei einem Projekt was "clean" in der Doku als Grund anführt ist das aber eine richtige Anmerkung ;-)